### PR TITLE
2.0 devel singleton refactor

### PR DESCRIPTION
**Pull request by japokorn**

### CODE REVIEW

**Comment by dwlehman** [comment]
You should remove the code that mocks the old devicetree attributes that held the cache data in the unit tests. It should be fairly easy to find, just before the same blocks where you had to mock/patch things.

File: blivet/populator/helpers/lvm.py [hunk 1]
```diff
@@ -37,6 +37,9 @@
 from .devicepopulator import DevicePopulator
 from .formatpopulator import FormatPopulator
 
+from ...staticdata.lvm_info import lvs_info, pvs_info
+
+
```
Comment (vpodzime) [inline blivet/populator/helpers/lvm.py]: Still two blank lines here. ;)

File: blivet/populator/populator.py [hunk 2]
```diff
@@ -45,6 +45,8 @@
 from ..threads import SynchronizedMeta
 from .helpers import get_device_helper, get_format_helper
 
+from ..staticdata.lvm_info import lvs_info, pvs_info
```
Comment (vpodzime) [inline blivet/populator/populator.py]: This could be just ``from ..staticdata import lvs_info, pvs_info``.

**Comment by vpodzime** [comment]
I think ``static_data`` would be a better ("more-PEP8") name for the module as it is composed of two words.

**Comment by japokorn** [comment]
Comments resolved ( also those mentioned in #345 ):
@dwlehman :
 
- Old devicetree variables removed from the test

@vpodzime :

- ``staticdata`` directory renamed to ``static_data`` (with all related code changes)
- Two blank lines in ``blivet/populator/helpers/lvm.py`` replaced with just one for (better) PEP8 compliance
- ``.lvm_info`` cut from the import line ``...staticdata.lvm_info import lvs_info, pvs_info`` in ``blivet/populator/populator.py``

**Comment by dwlehman** [comment]
It looks good to me now, but I think you should squash it into a single commit. I don't mind if you want to try letting github do that (if you read about that and are feeling brave) or just do it yourself (`git rebase -i origin/2.0-devel` to squash them together and then force-push to the PR branch before merging).

File: blivet/populator/helpers/lvm.py [hunk 3]
```diff
@@ -37,6 +37,8 @@
 from .devicepopulator import DevicePopulator
 from .formatpopulator import FormatPopulator
 
+from ...static_data.lvm_info import lvs_info, pvs_info
```
Comment (vpodzime) [inline blivet/populator/helpers/lvm.py]: ``lvm_info`` can be dropped here too.

**Comment by vpodzime** [comment]
Looks good to me now too other than the above nitpick. Squashing sounds like a good idea. I think no additional review is needed so just feel free to squash the commits and merge this PR.

**Pull request merged**